src/Forms/GridField/GridFieldDataColumns.php

```diff
@@ -154,7 +154,11 @@ public function getColumnContent($gridField, $record, $columnName)
     {
         // Find the data column for the given named column
         $columns = $this->getDisplayFields($gridField);
-        $columnInfo = $columns[$columnName];
+        $columnInfo = null;
+
+        if (array_key_exists($columnName, $columns)) {
+            $columnInfo = $columns[$columnName];
+        }
```

Comment:
How about a one liner like `$columnInfo = array_key_exists($columnName, $columns) ? $columns[$columnName] : null;`?

Reply:
@robbieaverill Changed it up.